tempesta_fw/http.c

```diff
@@ -1494,19 +1520,9 @@ tfw_http_conn_cli_drop(TfwCliConn *cli_conn)
 	 * condition with freeing of a request in tfw_http_resp_fwd().
 	 */
 	spin_lock(&cli_conn->seq_qlock);
-	list_for_each_entry_safe(req, tmp, seq_queue, msg.seq_list) {
-		if (req->resp)
-			list_move_tail(&req->msg.seq_list, &zap_queue);
-		else
-			list_del_init(&req->msg.seq_list);
-	}
+	list_for_each_entry_safe(req, tmp, seq_queue, msg.seq_list)
+		list_del_init(&req->msg.seq_list);
```

Comment:
There's the description to this function which describes the logic of actions that were removed. The description should be modified to conform to the current logic.